packages/data-stores/src/plans/types.ts

```diff
@@ -156,8 +113,11 @@ export interface Feature {
 	type: string;
 }
 
+export type DetailsAPIFeature = Omit< Feature, 'type' >;
```

Comment:
Doesn't it make more sense to let `Feature` inherit from `DetailsAPIFeature` and extend it with `type: string;` instead of the other way around? In that case, we don't have to omit props from an object. What do you thing?

Reply:
> What do you think?
Yep